docs/apis/api-gateway/adocs/admin_get/curl-request.adoc

```diff
@@ -0,0 +1,4 @@
+[source,bash]
```

Comment:
This is definitely a huge PR with 126 files change :-) How do we get those files generated? 

Is it possible that we only check in the codes that could trigger docs auto generation on the compile time, for example, when we do "mvn test & mvn install"